langchain/schema.py

```diff
@@ -23,4 +23,7 @@ class Generation(NamedTuple):
 
     text: str
     """Generated text output."""
+
+    finish_reason: Optional[str] = None
+    """Reason for finishing (OpenAI specific)"""
```

Comment:
if this is openai specific, lets just add a `generation_info` optional dictionary where providers can put provider specific information. that way this doesnt grow in size if we have a bunch of provider specific fields

we do this for LLMOutput: https://github.com/hwchase17/langchain/blob/6d78be0c8341baec4f9c5559db8c85d8956a7a74/langchain/llms/base.py#L20

Reply:
Makes sense. lemme update

Reply:
Made changes, lmk what you think